### PR TITLE
Start using embed module for assets

### PR DESCRIPTION
**Pull request by Fedosin**
This commit starts using "embed" module for static assets and removes the generated ones, created by go-bindata. This should help developers, as they don't have to regenerate bindata after each change in the assets.

### CODE REVIEW

File: bindata/assets.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package v410_00_assets
```
Comment (sttts) [inline bindata/assets.go:1]: do we still need this package name?

Comment (Fedosin) [reply bindata/assets.go:1]: it's mostly done for backward compatibility. If you think it is reasonable to change it to something else, I'm absolutely fine with it.

**Comment by Fedosin** [comment]
/hold

^ for tests to pass

**Pull request merged**